hw/top_earlgrey/dv/cov/clkmgr_tgl_excl.cfg

```diff
@@ -0,0 +1,16 @@
+// Copyright lowRISC contributors.
```

Comment:
I believe @tjaychen's approach was to automate this using our topgen flow. But this is fine with me too.

Reply:
I discussed this with @tjaychen: I'll commit these, and separately have the clkmgr exclusions here be generated via topgen.